src/DynamoCore/Logging/DynamoAnalyticsClient.cs

```diff
@@ -119,7 +121,7 @@ public DynamoAnalyticsClient(HostAnalyticsInfo hostAnalyticsInfo)
             Session.Start();
 
             //Dynamo app version.
-            var appversion = DynamoModel.AppVersion;
```

Comment:
is there a reason not to also remove this static property now?
[`DynamoModel.AppVersion`](https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCore/Models/DynamoModel.cs#L290)

Reply:
It has been removed, look at latest

Reply:
It seems to still exist on your branch - did you push all your changes?

https://github.com/BogdanZavu/Dynamo/blob/5fcc897fc1869f4eb510d10dfe0c1cce0e9bda25/src/DynamoCore/Models/DynamoModel.cs#L294C42-L294C42

Reply:
Ah, from model, yes, removed now